BRANCH_CLEANUP.md

```
@@ -0,0 +1,120 @@
+  coderabbitai/docstrings/25380cd \
+  coderabbitai/utg/4ce4e44 \
+  codex/deploy-tor-dns-service-with-uptime-monitoring \
+  codex/deploy-zk-lotto-and-encrypted-messenger \
+  feat/onion-auth feat/onion-captcha feat/onion-config-update feat/onion-crypto \
+  feat/onion-db-module feat/onion-middleware feat/onion-questions feat/onion-templates \
+  feat/site-nav-assets feature/completion-screen
+```
+
+Recovery (restore any): `git push origin <sha>:refs/heads/<name>`
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)